index.bs

```diff
@@ -1688,8 +1745,8 @@ ceremony, a [RP] MUST proceed as follows:
 
 13. If the attestation statement |attStmt| verified successfully and is found to be trustworthy, then register the new
     credential with the account that was denoted in the
-    {{WebAuthentication/makeCredential(options)/options}}.{{MakeCredentialOptions/user}} passed to {{makeCredential()}}, by
-    associating it with the credential ID and credential public key contained in |authData|'s [=attestation data=], as
+    {{ScopedCredential/[[Create]](options)/options}}.{{MakeCredentialOptions/user}} passed to {{CredentialsContainer/create()}},
+    by associating it with the credential ID and credential public key contained in |authData|'s [=attestation data=], as
```

Comment:
After the CM API merge, the implication of the current text is that the RP should take the credential ID directly from the attestation data, and ignore the id in the credential object itself. This seems like the right thing to do and actually a security measure. Should we make this explicit in the text here?